packages/api/tests/endpoints/test-rules.js

```diff
@@ -414,6 +599,11 @@ test('PUT replaces a rule', async (t) => {
     .set('Authorization', `Bearer ${jwtAuthToken}`)
     .expect(200);
 
+  const dbRecords = await dbClient.select()
+    .from(tableNames.rules)
+    .where({ name: expectedRule.name });
+
+  t.is(dbRecords.queue_url, undefined);
```

Comment:
I think this is passing for the wrong reason. You need to add a `first()` after the `.where` on the query above in order to get a DB record. Right now `dbRecords` is an array of DB results, on which `queue_url` definitely will not be defined

Reply:
Ah, nice catch. Another signifier is that the property should be coming back `null` instead of `undefined`. Thanks for pointing this out.